wikimedia/wikipedia-ios
# Update profile nav button accessibility labels

**Phabricator:** https://phabricator.wikimedia.org/T374167

### Notes
This PR updates the accessibility label for the profile button on Explore, Article and Settings. For wording, I followed the pattern of our old bell icon accessibility label ("Notifications" and "Notifications with unread badge") and used "Profile" or "Profile with unread badge".

### Test Steps
1. Run app from branch, be sure you are logged into an account with notifications in notifications center. Turn on Voice Over. Focus on profile badge on Explore, ensure it reads out "Profile" or "Profile with unread badge". Tap Profile button, navigate to Notifications, then do what is needed to toggle the Profile button state (either badged or unbadged). Go back to Explore feed and focus the profile button. Ensure it reads out "Profile" or "Profile with unread badge"
2. Repeat step 1 for Article profile button and Settings (AKA Explore feed disabled) profile button.
